templates/education/ecps-profiles.php

```diff
@@ -0,0 +1,159 @@
+$clone_office                      = ( isset( $profile_cct['clone_office']['0']['text'] ) ) ? $profile_cct['clone_office']['0']['text']: false;
+$research                          = ( isset( $profile_cct['research']['textarea'] ) ) ? $profile_cct['research']['textarea']: false;
+$clone_scholarly_interests         = ( isset( $profile_cct['clone_scholarly_interests_text_area']['textarea'] ) ) ? $profile_cct['clone_scholarly_interests_text_area']['textarea']: false;
+$research_interests                = ( isset( $profile_cct['clone_individual_research_interests']['0']['text'] ) ) ? $profile_cct['clone_individual_research_interests']['0']['text']: false;
+$address                           = ( isset( $profile_cct['address']['textarea'] ) ) ? $profile_cct['address']['textarea']: false;
+$website                           = ( isset( $profile_cct['website']['textarea']) ) ? $profile_cct['website']['textarea']: false;
+
+?>
+
+<div id="<?php echo absint( $post_id ); ?>" class="<?php echo esc_attr( $plain_term_slug ); ?> iso-profile">
+
+  <div class="boxey <?php echo esc_attr( $odd_even ); ?> <?php echo esc_attr( $plain_term_slug ); ?> profile">
+
+    <div class="boxey-inside">
+
+        <div class="boxey-inner">
+ 
+          <div class="iso-featured-image"><a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline"><img src="<?php echo esc_url( $post_thumbnail ); ?>" alt="" /></a></div>
+
+          <h3><div class="iso-title">
+
+            <a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline">
+            <?php if ( false !== $salutations ) : echo esc_html( $salutations ); endif; ?> <?php if ( false !== $first_name ) : echo esc_html( $first_name ); endif; ?> <?php if ( false !== $last_name ) : echo esc_html( $last_name ); endif; ?>          
+            </a>
+
+          </div></h3>
+
+          <div class="iso-description hidden">
+            <?php if ( false !== $salutations ) : echo esc_html( $salutations ); endif; ?> 
+            <?php if ( false !== $first_name ) : echo esc_html( $first_name ); endif; ?> 
+            <?php if ( false !== $middle ) : echo esc_html( $middle ); endif; ?> 
+            <?php if ( false !== $last_name ) : echo esc_html( $last_name ); endif; ?>
+            <?php if ( false !== $position ) : echo esc_html( $position ); endif; ?> 
+            <?php if ( false !== $clone_scholarly_interests ) : echo esc_html( $clone_scholarly_interests ); endif; ?> 
+            <?php if ( false !== $research_interests ) : echo esc_html( $research_interests ); endif; ?> 
+            <?php echo esc_html( $plain_term_name ); ?>
+          </div>
+
+          <?php if ( false !== $clone_room_number ) : echo esc_html( $clone_room_number ); endif; ?></br>
+
+          <?php if ( false !== $email ) :  echo wp_kses_post( "\n"."<i class='icon-envelope'></i> <a href='mailto:$email'>$email</a>"."\n" ) ;  endif; ?></a></br>
```

Comment:
This isn't quite right. If the $email is false, you'll still get the closing `</a>` tag.